frontend/src/utils.js

```diff
@@ -71,4 +71,27 @@ export const snakeToCamelCase = (str) =>
 
 const camelToSnakeCase = (str) => str.replace(/([A-Z])/g, "_$1").toLowerCase();
 
+export const storiesToGeoJson = (stories) => {
+  stories = stories.filter(
+    (story) => story && story.latitude && story.longitude
+  );
+
+  let features = stories.map((story) => {
+    let { latitude, longitude, ...properties } = story;
+    return {
```

Comment:
we could add a line for generating random number for lat&lon here, i.e. `latitude += random(); longitude += random();`

Reply:
Added